src/issues/issue-2/david-byrne/usePositions.js

```diff
@@ -0,0 +1,52 @@
+import { useTransition } from "react-spring"
+
+class NumberCruncher {
```

Comment:
A helper class to generate a random number between a range and make it positive or negative